f8a_worker/storages/s3_vulndb.py

```diff
@@ -56,3 +54,25 @@ def retrieve_depcheck_db_if_exists(self, data_dir):
                 Archive.extract_zip(archive_path, data_dir)
                 return True
         return False
+
+    def store_victims_db(self, data_dir):
+        """ Zip Victims CVE DB and store to S3 """
+        with tempdir() as archive_dir:
+            archive_path = os.path.join(archive_dir, self._VICTIMS_DB_ARCHIVE)
+            with cwd(data_dir):
+                try:
+                    Archive.zip_file(self._VICTIMS_DB_DIR, archive_path)
+                except TaskError:
+                    pass
+                else:
+                    self.store_file(archive_path, self._VICTIMS_DB_ARCHIVE)
```

Comment:
5. and this should end with an error, since your archive is stored inside `data_dir` folder